### PR TITLE
Gtk3 icon view initialize hash argument

### PR DESCRIPTION
**Pull request by cedlemo**

### CODE REVIEW

**Comment by kou** [comment]
Ah, sorry. I implemented another approach before I see this pull request...: Both `Gtk::IconView.new(model)` and `Gtk::IconView.new(area)` are acceptable.

Your approach (`Gtk::IconView.new(:model => model)` and `Gtk::IconView.new(:area => area)` API) is also good. Umm. Which approach should we choose...?

**Comment by cedlemo** [comment]
I let you choose even if I think that mine is more consistant with the previous api (like Gtk::ComboBox for example).

**Comment by kou** [comment]
Thanks for your opinion. Consistency is reasonable reason.

We choose your API. Could you this pull request rebase on the master?

**Comment by cedlemo** [comment]
Rebase done.

File: gtk3-no-gi/test/test_gtk_icon_view.rb [hunk 1]
```diff
@@ -1,6 +1,4 @@
-# -*- coding: utf-8 -*-
-#
-# Copyright (C) 2014  Ruby-GNOME2 Project Team
```
Comment (kou) [inline gtk3-no-gi/test/test_gtk_icon_view.rb]: Please keep start year.

File: gtk3-no-gi/test/test_gtk_icon_view.rb [hunk 2]
```diff
@@ -19,23 +17,23 @@
 class TestGtkIconView < Test::Unit::TestCase
   include GtkTestUtils
 
-  class TestGetCellRect < self
```
Comment (kou) [inline gtk3-no-gi/test/test_gtk_icon_view.rb]: Please don't remove the tests.

File: gtk3/lib/gtk3/icon-view.rb [hunk 3]
```diff
@@ -15,8 +15,20 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
 
 module Gtk
-  class IconView
-    alias_method :get_cell_rect_raw, :get_cell_rect
+  class IconView 
+    alias_method :initialize_raw, :initialize, :get_cell_rect_raw, :get_cell_rect
```
Comment (kou) [inline gtk3/lib/gtk3/icon-view.rb]: Two `alias_method`s should not be merged.
Please don't touch `alias_method :get_cell_rect_raw, :get_cell_rect`.

**Comment by kou** [comment]
Thanks. But there are some merge misses. Could you confirm them?

**Comment by cedlemo** [comment]
Yes I have failed the rebase. I am sorry I was in a hurry and I didn't really pay attention. Right now I don't have the time to fix this but I think that I can correct all the errors I have done within 4 or 5 hours.

I am very embarrassed for this kou.

**Comment by kou** [comment]
Never mind. :-)
Thanks for your many contributions. They are very helpful.

**Comment by kou** [comment]
Thanks! I've merged.

**Pull request merged**